src/components/AboutMeSection/AboutMeSection.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useStaticQuery, graphql } from 'gatsby';
+import Img from 'gatsby-image';
+import { rem } from 'polished';
+import { FormattedMessage } from 'gatsby-plugin-intl';
+
+import { breakPoints } from 'styles/base';
+import { SectionWrapper, SectionHeader } from '../Section/styled';
+
+const StyledImg = styled(Img)`
+  width: 90%;
```

Comment:
hm... it's weird that you are adding this width. Should be used padding on its wrapper